drivers/volume/ocp/ocp.go

```diff
@@ -101,6 +104,24 @@ func (o *ocp) InspectVolume(name string) (*api.Volume, error) {
 	}
 }
 
+// UpdateFBDANFSEndpoint updates the NFS endpoint for a given FBDA volume
```

Comment:
we support FA with OCP, can we also implement the functions here ?

Reply:
This is FBDA endpoint. It can be practically be on any environment, for the purpose of this test case, we don't need to do this.

Reply:
As answered by @pnookala-px this is not required .